i18n/ui/lbm-ui_en-US.po

```diff
@@ -527,8 +531,8 @@ msgid "Initializing hardware."
 msgstr "Initializing hardware."
 
 #: On screen: booting
-msgid "We'll be up in a moment."
-msgstr "We'll be up in a moment."
+msgid "<strong>We'll be up in a moment.</strong>"
```

Comment:
Having the tags on the ID should change what's translatable based on the locales.js file. Can you verify that this string gets translated to another language?